WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -62,6 +64,28 @@ struct OrderPaymentSection: View {
             .padding()
         }
     }
+
+    @ViewBuilder private var feesRow: some View {
```

Comment:
Do you know if there is a difference between a `@ViewBuilder var` and a `@ViewBuilder func`?  a function seems more appropriate to me.

Reply:
It was added in Swift 5.4. I don't see any functional differences or public guidelines.
Here I picked `var` approach because we don't pass any arguments and root `var body` is already declared same way:

```swift
@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
public protocol View {

    /// The type of view representing the body of this view.
    ///
    /// When you create a custom view, Swift infers this type from your
    /// implementation of the required ``View/body-swift.property`` property.
    associatedtype Body : View

    /// The content and behavior of the view.
    ///
    /// When you implement a custom view, you must implement a computed
    /// `body` property to provide the content for your view. Return a view
    /// that's composed of built-in views that SwiftUI provides, plus other
    /// composite views that you've already defined:
    ///
    ///     struct MyView: View {
    ///         var body: some View {
    ///             Text("Hello, World!")
    ///         }
    ///     }
    ///
    /// For more information about composing views and a view hierarchy,
    /// see <doc:Declaring-a-Custom-View>.
    @ViewBuilder var body: Self.Body { get }
}
```